internal/hcsoci/credentials.go

```diff
@@ -28,26 +28,27 @@ import (
 // setting up instances manually is not needed, the GMSA credential specification
 // simply needs to be present in the V1 container document.
 
-// CCGInstance stores the id used when creating a ccg instance. Used when
+// CCGResource stores the id used when creating a ccg instance. Used when
```

Comment:
Just curious if there was a particular reason for this name change?

Reply:
CCG instance is usually what's referred to when describing the actual instance passed around in HCS with all the relevant fields. CCGResource is just a type to signify that we've allocated one and would like to clean it up on container tear down. In CreateCredentialGuard if this wasn't renamed we'd be returning an  hcsschema.ContainerCredentialGuardInstance and a CCGInstance which feels odd and made naming the local variables in the caller annoying. Let me know if you have a case for the old name/if that's preferred to keep it how it was.

Reply:
That seems fair. I was just curious, no updates needed.